checks/fileparser/listing.go

```diff
@@ -76,10 +76,6 @@ func OnMatchingFileContentDo(repoClient clients.RepoClient, matchPathTo PathMatc
 	onFileContent DoWhileTrueOnFileContent, args ...interface{},
 ) error {
 	predicate := func(filepath string) (bool, error) {
-		// Filter out test files.
-		if isTestdataFile(filepath) {
-			return false, nil
-		}
```

Comment:
This is going to have changes beyond this check. E.g. we don't penalize test binaries in Binary-Artifacts.

If it makes sense to look at test files, I think it's better to have another function which does that. We can extract the common logic so we don't have two implementations.

Reply:
I wrapped it in a conditional, wdyt?